src/contracts/treasury/Collector.sol

```diff
@@ -35,7 +36,7 @@ contract Collector is VersionedInitializable, ICollector, ReentrancyGuard {
   /**
```

Comment:
cannot comment as the line was not changed, but `_fundsAdmin` is deprecated (at least i don't see it used anywhere where it makes sense, so it probably should be prefixed with `_deprecated` and documented on the natspec.

`getFundsAdmin` should be removed

An alternative would be to actually reuse the storage for `aclManager` - i think both are valid options.

Reply:
I opted towards deprecating rather than reusing the storage